src/public/modules/core/css/toolset-dropdown.css

```
@@ -0,0 +1,58 @@
+.toolset_dropdown_div {
+  position: relative;
+  display: -webkit-inline-box;
+  display: -ms-inline-flexbox;
+  display: inline-flex;
+  height: 55px;
+  float: right;
+}
+
+.toolset_dropdown {
+  cursor: initial;
+  z-index: 1000;
+  width: 280px;
+  position: absolute;
+  top: 55px;
+  -webkit-box-shadow: 1px 3px 5px 0 rgba(48, 51, 54, 0.2);
+  box-shadow: 1px 3px 5px 0 rgba(48, 51, 54, 0.2);
+  background-color: #fff;
+}
+
+.toolset_dropdown ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.toolset_dropdown ul li {
+  padding: 0px;
+  border-bottom: 1px solid #dcdcdc;
+}
+
+.toolset_dropdown_anchor_wrapper {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  padding-right: 24px;
+  padding-left: 24px;
+  padding-top: 18px;
+  padding-bottom: 18px;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+}
+
+.toolset_dropdown_anchor {
+  text-decoration-line: underline !important;
+  color: #2f60ce !important;
+  cursor: pointer;
+}
+
+.toolset_dropdown_list_anchor {
+  margin: 0px 0px 0px 0px !important;
```

Per our css linting rules, `0` do not need units
```suggestion
  margin: 0 0 0 0 !important;
```

---

done